app/controllers/svie_controller.rb

```diff
@@ -39,7 +39,7 @@ def hierarchy
 
   def application_pdf
     file_options = { filename: 'szerzodes.pdf', disposition: 'attachment', type: :pdf }
-    send_file('public/dowloads/contract_empty.pdf', file_options)
+    send_file("#{Rails.root}/public/downloads/contract_empty.pdf", file_options)
```

Comment:
```suggestion
    path         = Rails.root.join('public', 'downloads', 'contract_empty.pdf')

    send_file(path, file_options)
```
I think this way it should work

Reply:
Done